src/panels/lovelace/hui-root.ts

```diff
@@ -71,6 +71,15 @@ class HUIRoot extends LitElement {
     );
   }
 
+  protected firstUpdated(changedProperties: PropertyValues): void {
+    super.firstUpdated(changedProperties);
+    const userAgent = navigator.userAgent;
+
+    if (userAgent.match(/Android/) && userAgent.match(/Chrome/)) {
```

Comment:
```
    this.classList.toggle(
      "disable-text-select",
      /^(?=.*chrome)(?=.*android)/i.test(navigator.userAgent)
    );
```

Reply:
regex.test is way faster if you just want to know if it matches, use string.match when you want to get an array of all matching elements. (would still be ok to split it in 2 regexs, if that's more readable...

```
    this.classList.toggle(
      "disable-text-select",
      /Chrome/.test(navigator.userAgent) && /Android/.test(navigator.userAgent)
    );
```